test/e2e/utils/test_context.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	certmanagerVersion        = "v1.5.3"
+	certmanagerVersion        = "v1.13.3"
 	certmanagerURLTmpl        = "https://github.com/cert-manager/cert-manager/releases/download/%s/cert-manager.yaml"
```

Comment:
we meed to fix: https://github.com/cert-manager/cert-manager/issues/3495#issuecomment-1247395252